ee/codegen/src/generators/base-persisted-file.ts

```diff
@@ -206,8 +206,14 @@ export abstract class BasePersistedFile extends AstNode {
 
     let contents: string;
     try {
-      contents = await writer.toStringFormatted({ line_width: 120 });
-      contents = this.postprocessDocstrings(contents);
+      if (skipFormatting) {
+        // Sometimes we need to disable formatting because of fern problems, this isn't strictly necessary because
+        // of the catch statement below but it will let us avoid an unneeded error log and sentry report.
+        contents = writer.toString();
+      } else {
+        contents = await writer.toStringFormatted({ line_width: 120 });
+        contents = this.postprocessDocstrings(contents);
```

Comment:
[oos] a midterm approach here is to give ourselves some hook here for pre-processing before it's fed into the formatter